example/docs/introduction/getting-started.md

```diff
@@ -58,31 +58,36 @@ export default {
 ```
 ```css
 /*
-	Maybe you need to set the limits for the cropper sizes or its container sizes
-	otherwise a cropping image will try to fill all available space
+	Control the size of the cropper bounding box to prevent over-fill as the cropper
+	will atempt to fill the html element.
```

Comment:
I offer this variant:
> It may be necessary to set limits on the size of the cropper, otherwise the cropper image will try to fill all the available space.

Reply:
That sounds nice and clear to me.

Reply:
I added width because when I used it in my Vue application I needed to add the width because I was using the component inside a customized div so I needed o specify it. Also some people might not realize they can use it.

Reply:
Very well, it sounds reasonable.

Reply:
Yea I'm working on that. I had to pull the fork onto my terminal and I'm applying the updates right now.

Reply:
Done

Reply:
I've merged this pull request. Thanks.